application/image_caption_generator.py

```diff
@@ -1,18 +1,33 @@
+import utils
 from application.image_processor import ImageProcessor
 
 
-class ImageCaptionGenerator:
-    def __init__(self):
-        self.using_url: bool = False
-        self.image_processor: ImageProcessor
+def process_input(user_input: str) -> None:
+    ImageProcessor(user_input).process_input()
+    return
 
-    def start(self):
-        user_input: str = input('Will you use an image URL or a file directory? (url/dir)\n> ')
-        self.process_input(user_input)
 
-    def process_input(self, user_input: str) -> None:
-        self.using_url = True if user_input.lower().__contains__('u') else False
+def start():
+    print('Please provide an image URL(s) or a directory path to an image(s). If submitting '
+          'multiple URLs, separate them with a comma and a space (i.e, ", ").\n(You may quit the application at '
+          'anytime by entering "q" or "quit")')
+    user_input: str = input('\n> ')
 
-        image_processing = ImageProcessor(self.using_url)
-        image_processing.process_image()
+    if utils.quitting(user_input):
+        utils.end()
         return
+
+    process_input(user_input)
+
+    prompt_again()
+
+
+def prompt_again() -> None:
```

Comment:
Personal preference: Rename to repromt()
More readable and clear IMO

Reply:
Reprompt is more concise, but I didn't add it initially since it isn't a word. But this change was made.